action/sealedenvelope.go

```diff
@@ -134,8 +142,8 @@ func (sealed *SealedEnvelope) loadProto(pbAct *iotextypes.Action, evmID uint32)
 	if err != nil {
 		return err
 	}
-	encoding := pbAct.GetEncoding()
-	switch encoding {
+	sealed.encoding = pbAct.GetEncoding()
```

Comment:
even through errors are returned in line 151, 159, and 162, the encoding value of `sealed` has been modified. Unless we are not sure that the `sealed.encoding` won't be used any where else or the `sealed` instance won't be reused again.
Thus, we may want to introduce a local variable `encoding` here.

Reply:
reverted